pseudopotential_calculator/calculations/generic.py

```diff
@@ -1,6 +1,15 @@
+from __future__ import annotations
+
 from abc import ABC, abstractmethod
 from dataclasses import dataclass, field
-from typing import Self
+from typing import TYPE_CHECKING, Self
+
+if TYPE_CHECKING:
+    from ase import Atoms
+
+
+def repeat_cell(cell: Atoms, amount: tuple[int, int, int]) -> Atoms:
```

Comment:
maybe create a new atoms file next to castep.py to put this and other 'atom' methods in